pkg/subctl/cmd/verify.go

```diff
@@ -164,6 +164,9 @@ func checkValidateArguments(args []string) error {
 	if len(args) != 2 {
 		return fmt.Errorf("Two kubeconfigs must be specified.")
 	}
+	if strings.Compare(args[0], args[1]) == 0 {
```

Comment:
This can be bypassed if the user gives full-path to one of the kubeconfigs and just the filename (or relative path) to the other. To make it fool-proof, shall we compare the server IP address from the kubeconfigs?

@mkolesnik , @mangelajo @skitt @nyechiel thoughts?

Reply:
Or even if two kubeconfigs have the same contents. I think the best way to check this would be to check the `rest.Config` instances we load; but that might end up being much more complex than checking the files.

Reply:
@skitt - should the `rest.Config` be checked at the testing framework?
Or I can load the `rest.Config` in the `verify` cmd, inside `checkValidateArguments` ?

Reply:
I would check the configs in the test framework; otherwise, if the test framework changes in a way that would allow testing inside a single cluster, we’d also have to change `subctl` to match.

Reply:
Makes sense to compare the contents as well, but let's not try to get too carried away.
I think 1st check if the filename is the same (simple quick check) and if it isn't then check if the contents are the same.

If it were a security concern it'd make sense to check more thoroughly, but for sake of simplicity I think compare file names and then contents will give us enough protection from user mistakes.